sway-lib-std/src/b512.sw

```diff
@@ -27,10 +27,7 @@ impl From for B512 {
     }
 
     fn into(self) -> (b256, b256) {
-        (
-            (self.bytes)[0],
-            (self.bytes)[1],
-        )
+        ((self.bytes)[0], (self.bytes)[1], )
```

Comment:
If you're going to format these, do you mind removing the `,` at the end and reformat again? I think the formatter will then do the right thing and give you `((self.bytes)[0], (self.bytes)[1])`

Reply:
I can do that. I just hit format on the whole repo and a load of stuff wasn't formatted. I guess this is the first PR since the latest formatter release ?

Reply:
But if I change them in a way that contradicts the formatter, won't CI fail ?

Reply:
Currently CI won't fail. See https://github.com/FuelLabs/sway/issues/2917. Also, what I'm suggesting is removing the `,` and then running the formatter again - the formatter won't do bad things in that case. Anyways, none of this matters that much :D Totally up to you how you want to do with this 😄

Reply:
Alright I removed the commas as I was resolving the conflicts in the JSON files.